api/mapping/views.py

```diff
@@ -1890,3 +1914,71 @@ def scanreport_admin_page(request, pk):
         args["is_admin"] = False
 
     return render(request, "mapping/admin_scanreport_form.html", args)
+
+
+@login_required
+def scanreport_table_list_page(request, pk):
+    args = {}
+
+    scan_report = ScanReport.objects.get(id=pk)
+    scan_report_name = scan_report.dataset
+
+    args["scan_report"] = scan_report
+    args["scan_report_name"] = scan_report_name
+
+    return render(request, "mapping/scanreporttable_list.html", args)
+
+
+@login_required
+def scanreport_fields_list_page(request, sr, pk):
+    args = {}
+
+    scan_report_table = ScanReportTable.objects.select_related("scan_report").get(
+        id=pk, scan_report__id=sr
+    )
+
+    args["pk"] = pk
+    args["scan_report"] = scan_report_table.scan_report
+    args["scan_report_table"] = scan_report_table
+
+    return render(request, "mapping/scanreportfield_list.html", args)
+
+
+@login_required
+def scanreport_values_list_page(request, sr, tbl, pk):
+    args = {}
+
+    scan_report_field = ScanReportField.objects.select_related(
+        "scan_report_table", "scan_report_table__scan_report"
+    ).get(id=pk, scan_report_table=tbl, scan_report_table__scan_report=sr)
+    scan_report_table = scan_report_field.scan_report_table
+    scan_report = scan_report_field.scan_report_table.scan_report
+
+    args["pk"] = pk
+    args["scan_report"] = scan_report
+    args["scan_report_field"] = scan_report_field
+    args["scan_report_table"] = scan_report_table
+
+    return render(request, "mapping/scanreportvalue_list.html", args)
+
+
+@login_required
+def update_scanreport_field_page(request, sr, tbl, pk):
```

Comment:
Is this still required?

Reply:
It looks like this is the method using the new URL patterns, so I think so